src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/__tests__/UploadPhotos.jest.tsx

```diff
@@ -474,7 +474,7 @@ describe("UploadPhotos", () => {
       await waitFor(() => {
         expect(screen.getAllByTestId("photo-thumbnail-error").length).toEqual(1)
         expect(screen.getByTestId("photo-thumbnail-error")).toHaveTextContent(
-          "File format not supported. Please upload JPG, PNG or HEIC files."
+          "File format not supported. Please upload files with supported formats."
```

Comment:
Should we have a more extensive list of supported files that we can use here? (like https://github.com/artsy/force/pull/14233/files#diff-a05481bf8da07c405652aacab0976bcc614c5748277025c7eca1b1fe0f96152fR133)